pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -168,6 +169,13 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	if err != nil {
 		return reconcile.Result{}, errors.Wrapf(err, "fails to create InferenceServicesConfig")
 	}
+
+	// Reconcile cabundlesecret
+	caBundleSecretReconciler := cabundlesecret.NewCaBundleSecretReconciler(r.Client, r.Scheme)
```

Comment:
If we want to keep using `Secrets` over `ConfigMaps`, I'd suggest to have a way to turn off this reconciler.

Some people is very picky with regards to security. In other projects I've seen instances of people disliking broad privileges (with a strong emphasis around secrets), and requesting trimming down RBAC privileges to the essential ones.

Reply:
if we move to configmap, then this issue will be solved.

Reply:
Converted Secret to configmap